### PR TITLE
Change rimraf to del-cli

### PR DESCRIPTION
**Pull request by calebeno**
It looks like rimraf isn't well supported anymore.  Ran into some issues with it.  I switch it to del-cli which seems to be a more up-to-date package.  Also the shrinkwrap file I added last time looks like it was way wrong.  It had all of the dev dependencies in it.  Fixed that also.  Sorry for yet another PR.  All that said, the good news is that I installed it from my fork and ran some tests.  It looks like everything is working as expected!  Could you merge this and do an npm publish to rc.6?

### CODE REVIEW

**Comment by oe** [comment]
`rimraf ` works well for me, and it's widely used. What problem are you encountering?

**Comment by calebeno** [comment]
Yeah, I've used it before with no ill effect. Now that I think about it, it's probably more an issue with the shrinkwrap. I was getting an issue with a required opperating system mismatch error. Let me do some other testing on it with just the shrinkwrap changes and see if that's the root cause.

**Comment by calebeno** [comment]
It was definitely the shrinkwrap that was the issue.  It's fixed now.  I put rimraf back in.  This commit now should just fix the shrinkwrap and update a few packages.  If you do a squash merge it will just add one commit (there should be a dropdown option for it).  If things look good could you do a publish for rc.6?

**Comment by calebeno** [comment]
@evecalm Could you publish rc 6?  It should already be updated in the package.json

**Comment by oe** [comment]
I've published rc.6. I'm busy for wok these day, sorry for that.

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "prepublishOnly": "npm run build",
```
Comment (oe) [inline package.json:8]: npm script `prepublishOnly` has been removed.  `npm run build` before  publish package seems curt,  `npm run test` should be better if there must be a `prepublishOnly` script.

**Comment by calebeno** [comment]
Thank you very much!  That's fine about the `prebublishOnly`.  Just something I was trying out.  I'll pull rc6 into my application and let you know how things go.

**Comment by oe** [comment]
I've made a stable release of  `v1.0.0`,  also with some improvements. you may have a try 😄

**Pull request merged**